api/models/transformer.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2020 The Merlin Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package models
+
+import "github.com/google/uuid"
+
+// Transformer is a service for pre/post-processing steps.
+type Transformer struct {
+	ID                string    `json:"id"`
+	Enabled           bool      `json:"enabled"`
```

Comment:
Can you remove the `Enabled` field?

Reply:
I've answered this question here: https://github.com/gojek/merlin/pull/15#discussion_r513907426

Do you have a better suggestion regarding enabling our user to reuse their previous transformer config?

Reply:
Oh got it, I missed that reply